net/mdns-repeater/Makefile

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright (C) 2014 OpenWrt.org
```

Comment:
Don't add OpenWrt as copyright holder unless you're affiliated. Use your own name or omit it.